.github/workflows/cicd.yml

```diff
@@ -23,7 +23,7 @@
 name: 'Lint and Test'
 
 env: 
-  APP_NAME: uw_spotseeker
+  DIR_NAME: uw_spotseeker
```

Comment:
the new APP_NAME in settings.py was interfering with this and trying to run test-app/test.py, which doesn't exist

Reply:
it was easier to change this variable because it's only used here